locales/de_DE.json

```diff
@@ -2317,12 +2317,12 @@
         "profile-document-count": "{{count}} Dokument"
       },
       "creator": {
-        "share-details": "Teilen Sie {{providerName}} Ihre Daten mit",
-        "share-details-sub": "Geben Sie Ihre Nash-Kontodaten an {{providerName}} weiter, um Ihr Konto einzurichten.",
-        "name": "Vollständiger rechtlicher Name",
+        "share-details": "Teile {{providerName}} deine Daten mit",
+        "share-details-sub": "Gib deine Nash-Kontodaten an {{providerName}} weiter, um dein Konto einzurichten.",
+        "name": "Vollständiger Name",
         "first-name": "Vorname",
         "last-name": "Nachname",
-        "email": "E-Mail-Adresse",
+        "email": "Email-Adresse",
```

Comment:
Welche ist richtig? Das muss über das ganze Dokument auch standardisiert werden. 

@mucke12 Was denkst du?

Reply:
ja hab am anfang Email geschrieben und dann in anderen zeilen gesehen, dass E-Mail-Adresse geschrieben wurde.